corehq/apps/accounting/management/commands/suppress_duplicate_invoices.py

```diff
@@ -0,0 +1,85 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+from corehq.apps.accounting.interface import get_subtotal_and_deduction
+from corehq.apps.accounting.models import Invoice, CreditLine, FeatureType
+from corehq.util.dates import get_first_last_days
+
+
+class Command(BaseCommand):
+    help = 'Put credit back for duplicate invoices and suppress them'
+
+    def add_arguments(self, parser):
+        parser.add_argument('year', type=int, help='The year of the statement period.')
+        parser.add_argument('month', type=int, help='The month of the statement period.')
+        parser.add_argument('note', type=str,
+                            help='A note to be added when crediting back the amount')
+
+    def handle(self, *args, **kwargs):
+        year = kwargs['year']
+        month = kwargs['month']
+        note = kwargs['note']
+
+        start_date, end_date = get_first_last_days(year, month)
+
+        # Filter the Invoice objects based on the date_start
+        invoices_of_given_date = Invoice.objects.filter(date_start__range=(start_date, end_date))
```

Comment:
we should also filter by `is_hidden_to_ops=False`

Reply:
You're right! Fixed here: ecac1a31cf8492dcaddaad85f38583a1aea04df5